src/styles/components/multiple-form.less

```diff
@@ -92,6 +92,21 @@ base/forms.less
 
   .multiple-form-modal {
 
+    .form-row-element {
+
+      &.checkbox {
+        &:extend(.align-self-flex-end);
+
+        &.column-1 {
```

Comment:
Alright so I actually changed this so that it doesn't rely on the `column-1` class being present.